app/(main)/(drawer)/(apply-now)/apply-now/preview.tsx

```diff
@@ -22,22 +22,24 @@ const Preview = () => {
   const [errorMessage, setErrorMessage] = useState("");
   const [processedAttachments, setProcessedAttachments] = useState([]);
   const [rawAttachments, setRawAttachments] = useState([]);
+  const [fileAttachments, setFileAttachments] = useState([]);
+  const [answers, setAnswers] = useState<any[]>([]);
+  const [timeline, setTimeline] = useState<any>();
+  const [quotation, setQuotation] = useState<any>();
```

Comment:
can we use a specific type instead of any? A lot of any is not good for production. These will throw errors in production. Best to handle type bugs in development.